storage/fifocache/fifocacheSharded_test.go

```diff
@@ -251,6 +251,15 @@ func TestFIFOShardedCache_Clear(t *testing.T) {
 	assert.Zero(t, l, "expected size 0, got %d", l)
 }
 
+func TestFIFOShardedCache_CloseDoesNotPanic(t *testing.T) {
```

Comment:
this does not test the fact that the code does not panic. Either test the panic or rename the test

Reply:
Renamed.